### PR TITLE
dropdown now supports badge

### PR DESCRIPTION
**Pull request by JefMari**
for issue https://github.com/dev-onenetworkecommerce/aia/issues/134
- i extend the `min-width` of dropdown from `140` to `160`

Dropdown with badge
image

Dropdown with both icon and badge
image
- i also fix this thingy

screenshot from 2015-11-03 17 22 15

### CODE REVIEW

File: styles/components/_dropdown.scss [hunk 1]
```diff
@@ -7,9 +7,9 @@
   > .menu {
     position: relative;
     padding: ($padding-x / 2) 0;
-    min-width: 140px;
+    min-width: 160px;
     font-size: $font-size-base;
-    background: $brand-white;
+    background-color: $brand-white;
```
Comment (srph) [inline styles/components/_dropdown.scss:12]: Not sure why we need to change this line. But, OK :ok_hand:

**Comment by JefMari** [comment]
wait image incoming!!

File: styles/components/_dropdown.scss [hunk 2]
```diff
@@ -70,6 +70,17 @@
     background-color: $brand-gray-lt;
   }
 
+  > .menu > a .badge {
+    float: right; 
+    background-color: $brand-primary;
```
Comment (srph) [inline styles/components/_dropdown.scss:75]: This should be emerald, right? I think `.list-group` also extends `.badge.-emerald`. No?

Comment (JefMari) [inline styles/components/_dropdown.scss:75]: nope. it dont have yet

Comment (JefMari) [inline styles/components/_dropdown.scss:75]: but yeah this is wrong

Comment (srph) [inline styles/components/_dropdown.scss:75]: What do you mean haha

Comment (srph) [inline styles/components/_dropdown.scss:75]: NEVERMIND

**Pull request merged**